analyzers/tests/SonarAnalyzer.UnitTest/Helpers/ParameterLoaderTest.cs

```diff
@@ -213,5 +221,24 @@ private static SonarCompilationReportingContext CreateCompilationWithOption(stri
             var compilationContext = new CompilationAnalysisContext(compilation, options, _ => { }, _ => true, default);
             return new(AnalysisScaffolding.CreateSonarAnalysisContext(), compilationContext);
         }
+        private string GenerateSonarLintWithParametrizedRule(string ruleId, string key, string value)
+        {
+            var ruleParameters = new List<SonarLintXmlRule>()
+            {
+                new SonarLintXmlRule()
+                {
+                    Key = ruleId,
+                    Parameters = new List<SonarLintXmlKeyValuePair>()
+                    {
+                        new SonarLintXmlKeyValuePair()
+                        {
+                            Key = key,
```

Comment:
```suggestion
                            Key = ruleId,
```

Reply:
As the comment above, to my understanding the key is used to match the name of the parameter that needs to be passed to the rule (es. "isRegularExpression") and the value is the actual value that the user is passing (es. "true").

Reply:
It's the parameter value! 
Thanks!